### PR TITLE
Assignment7+8

### PR DESCRIPTION
**Pull request by NicholasJohnsonDev**
Teammate Luis Garcia @TechGlue

### CODE REVIEW

**Comment by Ashrom0** [comment]
I will review this

**Review by Ashrom0 — COMMENTED** [review]
General
The project should look like the image below ✔
The project should have an index.html file ✔
With an index.js file in a js folder ✔
And a site.css file in a css folder (along with other css files) ✔
There should be a <head>
With a favicon ✔
And a title ✔
The <body> should use the middle 70% of the page ✔

Menu Bar
The page should have a title/menu bar ✔
Use Font Awesome for the hamburger icon ✔
When clicked, the menu should appear under it ✔
The menu should have four buttons with creative names (the buttons don't need to do anything) ✔

Create a card with <div>s and CSS
Create a card from <div>s ✔
The card should have a header, body, and footer ✔
The CSS for this should be in the CSS file and work easily for another card. ✔

Show jokes on the card
Use the joke API from the first assignment to display a joke on a card  ✔
Use Axios from the CDN for the API call.
If the joke API returns an error display a message to try again in a few moments ✔
The punch line should not show immediately, but appear after 4 seconds ❌ 
There should be a button to get another joke ✔
There should be <a> tag that links to the joke api documentation page ✔
This <a> should be manually styled as a button ✔
Buttons should have a background that is a gradient ✔
All buttons (even the <a> that you made into a button) should have a hover effect ❌
Extra Credit
Do the following:

Make the buttons in the menu do something creative ❌
Rather than just making the menu appear, have it fade in and slide down. ❌
Have the page jump to 100% width on a smaller screen and 50% on a larger screen with 70% on a regular screen.

You guys done a good job with the assignment, just a couple of small problems, need to have a hover effect on buttons and the punchline for joke immediately appear instead of the requested 4 second wait.

File: Assignment7+8/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            <p id="setup"></p>
+            <p id="delivery"></p>
+            <p id="single-joke"></p>
+          </div>
+          <hr class="margin-0" />
+          <div id="cardFooter">
+            <button class="gradient white-text tall" onclick="fetchJoke();">
+              <span class="medium-text">Another Joke</span>
+            </button>
+            <a
```
Comment (dcox2005) [inline Assignment7+8/index.html:63]: There is no gradient background, and when you hover over it, it doesn't change.

**Review by dcox2005 — COMMENTED** [review]
Overall really well done.  There are a few things noted to change, but very well done.


General
The project should look like the image below ✔
The project should have an index.html file ✔
With an index.js file in a js folder ✔
And a site.css file in a css folder (along with other css files) ✔
There should be a <head> ✔
With a favicon ✔
And a title ✔
The <body> should use the middle 70% of the page ✔


Menu Bar
The page should have a title/menu bar ✔
Use Font Awesome for the hamburger icon (https://fontawesome.com/start) ✔
When clicked, the menu should appear under it ✔
The menu should have four buttons with creative names (the buttons don't need to do anything) ✔
Create a card with <div>s and CSS ✔
Create a card from <div>s ✔
The card should have a header, body, and footer ✔
The CSS for this should be in the CSS file and work easily for another card. ✔
Show jokes on the card ✔
Use the joke API from the first assignment to display a joke on a card (https://v2.jokeapi.dev/joke/Programming) ✔
Use Axios from the CDN for the API call. (https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js) ✔
If the joke API returns an error display a message to try again in a few moments ❌
The punch line should not show immediately, but appear after 4 seconds ✔
There should be a button to get another joke ✔
There should be <a> tag that links to the joke api documentation page ✔
This <a> should be manually styled as a button ✔
Buttons should have a background that is a gradient ❌
All buttons (even the <a> that you made into a button) should have a hover effect ❌

Extra Credit
Do the following:
Make the buttons in the menu do something creative ❌
Rather than just making the menu appear, have it fade in and slide down. ❌✔
Have the page jump to 100% width on a smaller screen and 50% on a larger screen with 70% on a regular screen. ❌✔


Fundamentals
Ensure that:
This is built with raw HTML, CSS, and JavaScript, no external libraries aside from Axios and a font library ✔
The index.html file should work when opened from a file explorer ✔
Keep all files in appropriate folders ✔
Have warning free HTML, CSS, and JavaScript ✔
Use as little css as possible ✔
No inline CSS ✔
Validate HTML with: https://validator.w3.org/nu/#textarea (or similar) ✔
Validate CSS with: https://jigsaw.w3.org/css-validator/#validate_by_input (or similar) ✔
No errors when loading the page ✔
Choose simplicity over complexity ✔

File: Assignment7+8/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          <i id="menuIcon" class="fa fa-bars"></i>
+          <i id="closeIcon" class="fa-solid fa-x"></i>
+        </button>
+      </nav>
+    </header>
+
+    <main>
+      <h1>The Joke Page</h1>
+      <hr />
+      <div id="card">
```
Comment (mmwoodfo) [inline Assignment7+8/index.html:47]: The issue with using an ID as opposed to a class, is that you wouldn't be able to add another div with the card style to this page as that ID should be unique.

**Pull request closed without merging**